### PR TITLE
add happi support

### PR DESCRIPTION
**Pull request by untzag**
Adding basic happi support for `OPADevice` and `DelayDevice`. Works on my machine :laughing:.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,42 @@
...
+```
+
+Dump all yaq devices into database.
+Note that if you do this again all devices with the same name will be rewritten.
+
+```bash
+$ yaqd list -f happi | happi update
+```
+
+Manually edit `device_class` field, putting in `wright.DelayDevice` and `wright.OPADevice` where appropriate. Also edit type field.
```
Comment (ksunden) [inline README.md:27]: Probably should include an example here...

File: README.md [hunk 2]
```diff
@@ -2,3 +2,42 @@
...
+```
+[DEFAULT]
+path = ~/.local/share/happi/db.json
+```
+
+Populate environment variable.
+
+```bash
+$ export HAPPI_CFG=~/.config/happi/happi.ini
+```
```
Comment (ksunden) [inline README.md:18]: Windows instructions?

**Comment by ksunden** [comment]
Implementation looks good, only real comments are on readme

**Comment by untzag** [comment]
I'm not interested in documentation that is that specific at the moment. I'm literally only trying to help you and I work with this @ksunden, none of this is long for this world....

**Pull request merged**